apps/avatax/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM node:22.11-alpine AS builder
+RUN apk update
+RUN apk add --no-cache libc6-compat git python3 make g++
+WORKDIR /app
+
+# Set up PNPM
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN npm i --global corepack@latest && corepack enable
```

Comment:
Do you need to install it and enable? I think in 22 you can do:

```
RUN corepack enable
```
this is at least what we do in https://github.com/saleor/apps/blob/main/.devcontainer/avatax/Dockerfile

Reply:
I see that it fixes issue with pnpm 10+ - we just updated to it so it needs to be included 👍🏻